rpc/cc/arduino/cli/commands/v1/core.proto

```diff
@@ -51,12 +51,12 @@ message PlatformDownloadReq {
 	string version = 4;
 }
 
-message PlatformDownloadResp {
+message PlatformDownloadResponse {
 	// Progress of the downloads of platform and tool files.
 	DownloadProgress progress = 1;
 }
 
-message PlatformUninstallReq {
+message PlatformUninstallRequest {
 	// Arduino Core Service instance from the `Init` response.
     Instance instance = 1;
```

Comment:
Can you please fix the indentation of the `Instance instance = 1;` all over the repo?

Reply:
I'm a bit surprised that this is not linted, it seems that formatting is not a concern for the linter https://docs.buf.build/lint-rules/#formatting.
BTW I fixed the indentation to 2 spaces as suggested from the [google's protobuf style guide](https://developers.google.com/protocol-buffers/docs/style#standard_file_formatting) and also made the same comments style on all our files.

Reply:
>to our knowledge, no such formatter exists in the OSS world

Hmm. I was dreaming of eventually getting formatting set up. I identified clang-format as a candidate tool (it lists Protobuf on the supported languages list). Any thoughts on that?

Reply:
If it works, why not...